cmsdb/campaigns/run3_2022_postEE_nano_v12/higgs.py

```diff
@@ -1205,3 +1205,45 @@
         ),
     ),
 )
+
+####################################################################################################
+#
+# tH
+#
+####################################################################################################
+
+cpn.add_dataset(
+    name="thq_4f_madgraph",
+    id=15043210,
+    processes=[procs.thq],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/THQ_ctcvcp_HIncl_M-125_4FS_TuneCP5_13p6TeV_madgraph-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM",  # noqa: E501
+            ],
+            aux={
+                "broken_files": [],
+            },
+            n_files=127,  # 127-0
```

Comment:
I guess the `# 127-0` comment comes from a validation script, but here it might lead to questions in the future. Would you be up for removing the comments? (here and below).